min_tilt_version/Tiltfile

```diff
@@ -0,0 +1,24 @@
+def min_tilt_version(min_version):
+  # this assumes that the tit version command always returns
+  # v0.13.0, built 2020-04-01
+
+  ver_string = str(local('tilt version', quiet=True))
+  # Clean out the version file so its in the format 0.13.0
+  versions = ver_string.split(', ')
+  version = versions[0][1:]
+
+  # Python allows you to compare two tuples
+  # (0, 12, 0) < (0, 12, 1)
```

Comment:
I never knew this!

Reply:
Me too!! Every one keeps referring to pip package but I managed to find a native version that supported by skylark :)